modules/core/src/main/java/io/fluo/webindex/core/DataUtil.java

```diff
@@ -106,14 +109,36 @@ public static String toUri(URL url) {
   }
 
   public static String cleanUrl(String url) {
-    String cleanUrl = url.trim();
-    if (cleanUrl.length() >= 8) {
-      if (cleanUrl.substring(0, 7).equalsIgnoreCase("http://")) {
-        cleanUrl = "http://" + cleanUrl.substring(7);
-      } else if (cleanUrl.substring(0, 8).equalsIgnoreCase("https://")) {
-        cleanUrl = "https://" + cleanUrl.substring(8);
+    StringBuilder cleanUrl = new StringBuilder();
```

Comment:
Is the following the only cleaning this method does?

 * Strips `:` when nothing follows it in hostname